### DISCUSSION TITLE
Missing of the "Comboburst" feature in osu! lazer

### DISCUSSION THREAD

**Discussion by KirisvielYAYY** [ppy/osu#30354] [Ideas]
As a player, I really enjoy having comboburst on while playing and when I tried to switched to lazer not having them seemed like something was missing from the gameplay and I culd't enjoy it to much becouse of that.
I personally think lazer is better than stable in any way possible beside preference things which are more becouse I got use to std than the fact that they are better in stable, but the fact that comboburst are missing is reaaly a huge thing for me and the only reasno why I'm not switching to lazer

**Comment by Tenexxt** [comment]
As mentioned in the [last months lazer dev update video](https://www.youtube.com/watch?v=aFRwwCxRAuU) on minute 4:10, this is getting worked on and will be implemented in some time (ETA not included)

The issue regarding this can be found [here](https://github.com/ppy/osu/issues/4629) and i couldn't find an open pr for combo burst implementation exept for [this draft](https://github.com/ppy/osu/pull/17616).

_If you want this faster then i recommend looking for more people wanting this, the higher the players intrest, the higher will be the chance of devs focusing on it_
